content.md

```diff
@@ -700,6 +688,10 @@ the cards take precedence.
 
 ### **Wording templates**
 
+You may come across cards which use certain templates (e.g. Govern the
+Unaligned). For a better understanding, the most important templates are
```

Comment:
I'm not sure why the example? Maybe just remove the parenthesis? Or is that meant to mean an image should be added or something?

Reply:
Imo, there should be a small introduction for someone who doesn't know and I found an example helpful. A card image could be good imo - not sure atm why I chose Govern..